backend/pubsub.js

```diff
@@ -323,8 +328,16 @@ const publicMethods = {
     const server = this
 
     for (const client of to || server.clients) {
+      const msg = typeof message === 'string' ? message : JSON.stringify(message)
+      if (client.endpoint) {
+        console.error('@@@@POSTING PUSH EVENT')
```

Comment:
Please scan this PR for debug logging like this (also see one above on line 214) and remove it or change it to the appropriate log level.